app/assets/stylesheets/utilities/_flex.scss

```diff
@@ -17,6 +17,12 @@
   flex-wrap: nowrap;
 }
 
+.md\:flex-nowrap {
```

Comment:
I did not even know you could escape `:` in selectors. Please don't.

Reply:
That's how all utility classes work, it's inspired by Tailwind, which uses the same format

Reply:
Tailwind is clearly wrong, but I guess that's an entire community + you vs. me. I don't stand a chance.

Reply:
I don't actually like Tailwind, but I do like that format :D